wgsl/index.bs

```diff
@@ -1,10 +1,12 @@
 <pre class='metadata'>
 Title: WebGPU Shading Language
-Shortname: WGSL
-Level: 1
+Shortname: wgsl
```

Comment:
is it necessary to write this in lower-case?

Reply:
That is not a requirement. I'm following the practice used in most other specs but there are some exceptions-to-the-rule such as [SRI](https://www.w3.org/TR/SRI/), [CSS](https://www.w3.org/TR/CSS22/) (but most other CSS specs use lower-case shortnames), or [WebCryptoAPI](https://www.w3.org/TR/WebCryptoAPI), so it's possible to use upper-case if the group prefers that.

Reply:
This was discussed in the WGSL meeting of [2021-05-11](https://docs.google.com/document/d/1go1W35-B2lCuvbgFlfbmQstboU11XlWdlcn336ysBkg/edit#heading=h.80ppz6n3dozg)
The group agreed to keeping it "WGSL".    We can do that before or after landing this PR.